### PR TITLE
org clean up

### PR DESCRIPTION
**Pull request by teragopher**

### CODE REVIEW

File: map_org_names.yaml [hunk 1]
```diff
@@ -14,7 +14,7 @@ mappings:
   - ['^[[:space:]]*absa[[:space:]]*group[[:space:]]*$', 'Absa Group Limited']
   - ['^[[:space:]]*absolute[[:space:]]*softare[[:space:]]*$', 'Absolute Software Corporation']
   - ['^[[:space:]]*accenture[[:space:]]*india[[:space:]]*pvt[[:space:]]*ltd[[:space:]]*$', 'Accenture Ltd.']
-  - ['^[[:space:]]*accord[[:space:]]*project[[:space:]]*$', 'Accord Project, LLC']
+  - ['^[[:space:]]*(accord[[:space:]]*project|Accord)[[:space:]]*$', 'Accord Project, LLC']
```
Comment (lukaszgryglicki) [inline map_org_names.yaml:17]: RE is case insensitive, so please use only lower case letter in RE. Please apply to all other changes (MariaDB database is also case insensitive)

**Review by lukaszgryglicki — CHANGES_REQUESTED** [review]
Generally lgtm but please address my one comment. Thanks!

File: map_org_names.yaml (hunk 1 above)
Comment (teragopher) [reply map_org_names.yaml]: @lukaszgryglicki I have fixed this.. Thanks :)

**Comment by lukaszgryglicki** [comment]
Deployed on both `test` and `prod` - sorry for the delay but I was traveling.

**Pull request merged**